### PR TITLE
Implemented Pay System Alert

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#4795

*Description of changes:*

* Implemented common component `PaySystemAlert` to monitor and display issues with PAYBC on the entities dashboard.

*Notes: *

* This work requires OS config updates for dev/test/prod for the PayBc status url. This will be done immediately, and should be complete before merge.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/utils/fetch-config.ts [hunk 1]
```diff
@@ -31,6 +31,17 @@ export async function fetchConfig (): Promise<void> {
...
+  /**
+   * authConfig is a workaround to fix the user settings call as it expects a URL with no trailing slash.
+   * This will be removed when a fix is made to sbc-common-components to handle this
+   */
+  const authConfig = {
+    'VUE_APP_STATUS_ROOT_API': response.data['VUE_APP_STATUS_ROOT_API']
+  }
+  const authConfigString = JSON.stringify(authConfig)
+  sessionStorage.setItem('AUTH_API_CONFIG', authConfigString)
+  console.log('AUTH_API_CONFIG: ' + authConfigString)
```
Comment (lekshmimallika-aot) [inline src/utils/fetch-config.ts:43]: Should be info

Comment (cameron-freshworks) [reply src/utils/fetch-config.ts:43]: oo right, i saw these updates earlier! Will update in next commit 👍

File: public/config/configuration.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
   "LEGAL_API_URL": "https://legal-api-dev.pathfinder.gov.bc.ca/api/v1/",
   "AUTH_API_URL": "https://auth-api-dev.pathfinder.gov.bc.ca/api/v1/",
   "PAY_API_URL": "https://pay-api-dev.pathfinder.gov.bc.ca/api/v1/",
+  "VUE_APP_STATUS_ROOT_API": "https://status-api-dev.pathfinder.gov.bc.ca/api/v1",
```
Comment (severinbeauvais) [inline public/config/configuration.json:9]: Add a comment to the release ticket (/bcgov/entity#4672) to remember to update the config maps.

Comment (cameron-freshworks) [reply public/config/configuration.json:9]: Copy that, will do!

Comment (cameron-freshworks) [reply public/config/configuration.json:9]: This has been updated in Dev And Test, currently 👍

**Pull request merged**